src/mobile/apps/show/test/helpers/view_helpers.test.coffee

```diff
@@ -0,0 +1,33 @@
+sinon = require 'sinon'
+Show = require '../../../../models/show'
+{ fabricate } = require 'antigravity'
+ViewHelper = require '../../helpers/view_helpers.coffee'
+
+describe 'toJSONLD', ->
+  context 'when show has location', ->
+    show = new Show fabricate 'show'
+    jsonLD = ViewHelper.toJSONLD(show)
+    it 'jsonld has the show name', ->
```

Comment:
#superminor but it can help readability of these files a lot to put whitespace before these `it` blocks!